microcosm_flask/conventions/swagger.py

```diff
@@ -35,7 +34,7 @@ def operations(self):
         def match_func(operation, ns, rule):
             # only expose endpoints that have the correct path prefix and operation
             return (
-                rule.rule.startswith(make_path(self.graph, ns.path)) and
+                rule.rule.startswith(make_path(self.graph, swagger_ns.path)) and
```

Comment:
This was a bug; during the convention refactor, I accidentally switched which namespace was being used to compute whether an endpoint should be advertised with swagger.